modules/aggregation-layers/src/utils/aggregation-operation-utils.js

```diff
@@ -77,3 +90,20 @@ export function getValueFunc(aggregation, accessor) {
       return null;
   }
 }
+
+function wrapAccessor(accessor, context = {}) {
```

Comment:
Nit: Not a huge fan of this wrapping business - the explosion of small objects can get quite expensive memory wise for big datasets. Already felt an itch to see if it could be removed in geojson layer :)

I assume a separate typed array with Uint32Array with indices wouldn't work? It wouldn't be accessible where needed?

Reply:
`context` needs to be populated before calling `accessor`. The wrapping can arguably be moved outside of the loop so that it is done only once instead of for each cell. That would require significant change to the existing code base though.